File: java-examples/src/main/java/examples/hello.java
package examples;
import java.sql.*;
public class hello {
	public static void main(String[] args) {
		try (Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5432/testdb", "zemian", "test123")) {
            System.out.println("Connection successful!");
            System.out.println("Connection object=" + conn);
        } catch (Exception e) {
            throw new RuntimeException("Failed to connect DB", e);
        }
	}
}

File: php-examples/hello.php
<?php

// A simple DB connection test
echo "Connection Test: ";
$conn = pg_connect("host=localhost dbname=testdb user=zemian password=test123")
    or die('Could not connect: ' . pg_last_error());
echo "Successful!\n";

echo "Query Test:\n";
$result = pg_query('SELECT 1 + 1') or die('Query failed: ' . pg_last_error());
$rows = pg_fetch_array($result, null, PGSQL_ASSOC);
var_dump($rows);

pg_free_result($result);
pg_close($conn);
